doc/domains.rst

```diff
@@ -76,7 +76,7 @@ directive name.
 
 .. rubric:: Default Domain
 
-To avoid having to writing the domain name all the time when you e.g. only
+To avoid having to write the domain name all the time if you only
 describe Python objects, a default domain can be selected with either the config
```

Comment:
how about this:

   ... if one is for example only describing Python objects, ...

Also, I realize earlier paragraphs use the "you" form, which is perhaps not optimal for documentation. But this is other matter.